modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,71 @@
+You can then deploy them later in the game -- when they arrive, you disembark them into the play area and have them through a unit -- or objective -- that has this ability.
+====
+
+== Portal Reserves
+
+Where your army includes any units -- or objectives -- that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that can use a {portal} off-table.
+** Where a unit data sheet states `{portal}` and then `_Auto_` in brackets, detachments in {portal-reserves} do not follow the standard rules for Reserves.
+Instead, from turn two onwards, you must *must* disembark them from a {portal} in the Movement phase unless there are not enough Portals available, or none of the available Portals can disembark all the units in the detachment.
+** Where a unit data sheet states `{portal}` and then `_Roll_` in brackets, detachments in {portal-reserves} follow the standard rules for Reserves.
+When a detachment arrives from {portal-reserves}, you *must* disembark from a {portal} if possible.
+If there are not enough Portals available, or none of the available Portals can disembark all the units in the detachment, the detachment stays in {portal-reserves} and you must roll again next turn.
+** If all {portal} units/objectives in your army have been destroyed, or the game has ended, then all of your detachments still in {portal-reserves} are destroyed.
+** If the only Portals available cannot disembark all the units of a detachment in {portal-reserves}, you can choose to disembark the detachment anyway. The units that cannot disembark are immediately destroyed, and the detachment gains a Blast marker.
+
+== Portal
+
+Where your unit or objective has the {portal} ability:
+
+* Detachments that you place in {portal-reserves} *must* move onto the table through a {portal}.
+* When you wish to move the units in a detachment out of a portal, use the normal disembarkation rules as at xref:main-rules:transportation.adoc[].
+* When you disembark a detachment from a portal, you may choose only Advance or Assault orders.
+// TODO: Can you use any order you like when you disembark immediately after embark elsewhere in the play area as mentioned in the point a little further down?
+* You may only use each portal to embark or disembark a single detachment in each turn.
+* In the Movement phase, you may embark friendly units 'into' a portal -- but then you *must* immediately disembark them from a different {portal} elsewhere in the play area.
+* Data sheets state the units that can embark/disembark through the portal in brackets, for example:
+** `_(INF)_` allows Infantry units to use the {portal}.
+** `_(Walker)_` allows Vehicle units that have the {walker} special ability to use the {portal}.
+** `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
+** Often, we combine these.
+For example: `_(INF, {walker}, 4+, Auto)_`.
```

Comment:
This seems a bit strange to me: First, it seems illogical that units that didn't participate in the battle are automatically destroyed just because they are running late. 'Count as destroyed' for some game purposes would be one thing, but 'are destroyed' is another. Second, I wonder what you intend for Army Morale -- if all portals are destroyed, and thus all remaining Portal Reserves are destroyed, I believe this implies that we must deduct their Morale points at the end of the next Rally phase -- is this correct/what you intend?